### PR TITLE
feature #130: support multiple manifest file extensions 

### PR DESCRIPTION
**Pull request by h2non**
Tests are basic. Tries to follow your existent code and interface conventions.

Needs a review and probably some extra tests. Take a look and let me know.

### CODE REVIEW

File: cmd/yaml.go [hunk 1]
```diff
@@ -33,6 +38,46 @@ func ParseYaml(c cookoo.Context, p *cookoo.Params) (interface{}, cookoo.Interrup
 	return cfg, nil
 }
 
+// LookupManifestFile look up for a given file
+// in the current working directory
+func LookupManifestFile(fname string, p *cookoo.Params) (interface{}, cookoo.Interrupt) {
```
Comment (technosophos) [inline cmd/yaml.go]: So the way I read this, if I do `-y foo.yml` and I have two files, one named `foo.yml` and one named `foo.yaml`, then `foo.yaml` will get loaded even though I explicitly said `foo.yml`, right?

File: glide.go [hunk 2]
```diff
@@ -319,8 +319,8 @@ Example:
 		{
 			Name:      "update",
 			ShortName: "up",
-			Aliases:   []string{"install"},
-			Usage:     "Update or install a project's dependencies",
+			//Aliases:   []string{"install"},
```
Comment (technosophos) [inline glide.go]: Please leave the alias. We have it here for backward compatibility.

**Comment by technosophos** [comment]
To speed up dev on this, I did a quick ack of the codebase for references to `glide.yaml`.

```
$ ack 'glide\.yaml' ./cmd/*.go
cmd/flatten.go
176:	gp := path.Join(dir, "glide.yaml")
193:	Info("Found glide.yaml in %s", gp)

cmd/get_imports.go
67:			Warn("Package %q is already in glide.yaml. Skipping", root)

cmd/init_glide.go
33:// Among other things, it creates a default glide.yaml.
36:// 	- filename (string): The name of the glide YAML file. Default is glide.yaml.
39:	fname := p.Get("filename", "glide.yaml").(string)

cmd/link_package.go
29:		return nil, fmt.Errorf("glide.yaml is missing 'package:'")

cmd/util.go
37:// Most importantly, it fails if glide.yaml is not present in the current
40:	fname := p.Get("filename", "glide.yaml").(string)

cmd/vendor_path.go
14:	filename := c.Get("yaml", "glide.yaml").(string)
20:	// Find the directory that contains glide.yaml

cmd/vendor_path_test.go
11:	filename := "glide.yaml"

cmd/yaml.go
14:// ParseYaml parses the glide.yaml format and returns a Configuration object.
22:	fname := p.Get("filename", "glide.yaml").(string)
102:			Warn("Package %s is already in glide.yaml. Skipping.\n", dep.Name)
```

File: cmd/yaml.go (hunk 1 above)
Comment (h2non) [inline cmd/yaml.go]: I've just pushed a new commit which should look first at `.yaml`, and then to `.yml`, so now you're right. Having multiple files with the same name a different extension is an edge scenario and a mistake by the end user.

File: glide.go (hunk 2 above)
Comment (h2non) [inline glide.go]: Done!

**Comment by mattfarina** [comment]
This PR was incomplete and is no longer compatible with the current codebase so I'm going to close it. If someone wants to pursue this change we're happy to discuss it. Closing due to the state and it's comparison to master.

**Pull request closed without merging**